hieradata/site/tu/cluster/comcam-ccs.yaml

```diff
@@ -0,0 +1,10 @@
+---
+ccs_site: "tucson"
+
+## FIXME this should be "comcam-mcm-dds", same as real comcam.
+ccs_sal::dds_interface: "140.252.147.52"
```

Comment:
The `dds` interface still hasn't shown up in foreman for some reason.  I'll sort this out tomorrow.